module/documents/active-effect.mjs

```diff
@@ -391,6 +391,26 @@ export default class ActiveEffect5e extends ActiveEffect {
     return Promise.all(Array.from(riders).map(createRider));
   }
 
+  /* -------------------------------------------- */
+
+  /**
+   * Create additional effects that are applied separately from an enchantment.
+   */
+  async createRiderEnchantments() {
+    const origin = await fromUuid(this.origin);
+    const riders = (this.getFlag("dnd5e", "enchantment.riders") ?? []).map(id => {
+      const effectData = origin.effects.get(id)?.toObject();
+      if ( effectData ) {
+        delete effectData._id;
```

Comment:
Should we also set `disabled: false` here too, for good measure?

Reply:
Well, you might want to have effects that are added but aren't immediately enabled, like an enchanted item that you activate and then it applies an effect.